embark/users/views.py

```diff
@@ -1,11 +1,14 @@
 import logging
```

Comment:
users/views.py:126:20: R1705: Unnecessary "else" after "return" (no-else-return)
since you are using a else-return pattern for your if-clauses

`# pylint: disable=R1705`
should be added in line 1 before the imports